geomstats/geometry/lie_algebra.py

```diff
@@ -12,14 +12,14 @@
 import abc
 
 import geomstats.backend as gs
-import geomstats.errors
-from geomstats.geometry.base import VectorSpace
+from geomstats.errors import check_integer
+from geomstats.geometry.base import MathStruct
 from geomstats.geometry.matrices import Matrices
 
 from ._bch_coefficients import BCH_COEFFICIENTS
 
 
-class MatrixLieAlgebra(VectorSpace, abc.ABC):
+class MatrixLieAlgebra(MathStruct, abc.ABC):
```

Comment:
The LieAlgebra is the space equipped with the LieBracket.
If this inherits from a Structure, it needs to have the name of a structure, i.e., a LieBracket.